java/src/main/java/HelloWorldTests.java

```diff
@@ -0,0 +1,79 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.nio.file.Paths;
+
+
+public class HelloWorldTests {
+    private WebDriver driver;
+
+    @BeforeTest
+    public void setUp(){
+        String chromeDriverPath = Paths.get(System.getProperty("user.home"), "drivers", "chromedriver").toString();
+
+        // Optional, if not specified, WebDriver will search your path for chromedriver.
+        System.setProperty("webdriver.chrome.driver", chromeDriverPath);
+    }
+
+    @BeforeMethod
+    public void prepareWebDriver(){
+        driver = new ChromeDriver();
+    }
+
+    @Test(description = "Verify successful captcha input")
+    public void verifyCaptchaTest(){
+        driver.get("https://www.ultimateqa.com/filling-out-forms/");
+
+        //Find name field on the right side
+        WebElement rightNameInputElement = driver.findElement(By.id("et_pb_contact_name_1"));
+        //TODO Replace with your own name
+        //Enter your name
+        rightNameInputElement.sendKeys("Your NAME");
+
+        //Find message field on the right side
+        WebElement rightMessageInputElement = driver.findElement(By.id("et_pb_contact_message_1"));
+        //Enter "Hello World!"
+        rightMessageInputElement.sendKeys("Hello World!");
+
+        //Find equation text
+        String equationText = driver.findElement(By.className("et_pb_contact_captcha_question")).getText();
+
+        String numbers[] =  equationText.split("\\s*\\+\\s*");
+        //Add two numbers
+        int total = Integer.parseInt(numbers[0]) + Integer.parseInt(numbers[1]);
+
+        //Find answer field
+        final String ANSWER_INPUT_XPATH = "//input[contains(@class, 'input et_pb_contact_captcha')]";
+        WebElement answerInputElement = driver.findElement(By.xpath(ANSWER_INPUT_XPATH));
```

Comment:
Why did you use By.xpath here instead of using By.className? Just to demonstrate the use of XPath?

Reply:
Your comments brought a good question.Though best practices is use xpath as the last resort, but also this is a way to demonstrate how you can access WebElement. Hmm What do you think???

Reply:
changed to CSS